pkg/ref/expr_test.go

```diff
@@ -38,7 +38,8 @@ func TestResolve(t *testing.T) {
 			ObjectMeta: metav1.ObjectMeta{
 				Name: "app-namespace",
 				Labels: map[string]string{
-					labels.AcornAppName:      "app-name",
+					labels.AcornAppName: "app-name",
+					// Does the below not need to match "app-namespace"?
```

Comment:
The answer to this question is no. The namespace is called `app-namespace` and it corresponds to the `project-name`, which is the namespace where the `App` object lives.

Let's remove this comment.

Reply:
Perfect, thank you for the info! Removed